README.md

```diff
@@ -236,6 +236,25 @@ The property `explicitFolding.notification` (`minor` by default) indicates when
     "beginRegex": " \\{\\s*$",
     "endRegex": "^\\s*\\}"
 }</code>
+</pre>
+            </td>
+        </tr>
+        <tr>
+            <th>Helm</th>
+            <td>
+<pre>
+<code>"helm": [
+    {
+        "beginRegex": "\\{\\{-?\\s*if",
+        "middleRegex": "\\{\\{-?\\s*else",
+        "endRegex": "\\{\\{-?\\s*end"
+    },
+    {
+        "beginRegex": "\\{\\{-?\\s*(range|with|define|block)",
```

Comment:
@ohemelaar  Why `if` is separated? It could just be `(range|with|define|block|if)`